svg/compiler/test99.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What does the circle with the lines in it do?

Reply:
More wip stuff.  I needed to experiment to see how Draw.IO represents Rects, Text and Ellipses (three different formats, now represented in calc_bounds.pl).